test/exchangeFullOpsMainnet.ts

```diff
@@ -500,6 +500,44 @@ async function setupYCrvLpMinorCoin() {
     console.log("Minor coin (yCRV-f) is set.");
 }
 
+let frxEthLp: IERC20Metadata;
+async function setupfrxEthLpMinorCoin() {
```

Comment:
As we add both tokens in 1 step, could you please combine them in 1 function? Remember to register both adapters and routes in single function call